### DISCUSSION TITLE
How to interpret categorical coefficients in CoxPH Fitter

### DISCUSSION THREAD

**Discussion by JoCouweleers** [CamDavidsonPilon/lifelines#1467] [General]
Hi there, I'm doing a churn analysis with the Cox PH Fitter. My duration is tenure of the subscription, and the event is the churn itself. I'm including features like Product and Payment Model (discount vs. fully paid) which are categorical in nature vs. continuous variables. What's the best way to interpret the model coefficients in this case? Right now I have them included as dummy variables (dropping one for baseline). 

This verbiage makes sense to me for a continuous variable but struggling to apply this to a categorical one... "A one unit increase in "x" from its median value -> the hazards change by the factor of exp(coef) = Y, which is a % increase compared to the baseline hazards. 

input:
![image](https://user-images.githubusercontent.com/22525672/192376148-c043fcbb-44ce-4acb-88f4-07547057c634.png)

output: 
![image](https://user-images.githubusercontent.com/22525672/192376238-f5ace46f-16db-400e-959f-f31f178760f8.png)

**Comment by CamDavidsonPilon** [comment]
Does [this help](https://en.wikipedia.org/wiki/Proportional_hazards_model#A_single_binary_covariate)? I'm not being sarcastic by linking to wikipedia - those are new additions to the article that I wrote! After reading that, what questions do you have?

**Reply by JoCouweleers** [reply]
Thank you, that does help! Here I am scouring stackoverflow and the answer is sitting right on wikipedia.
